ui/widgets/nuxeo-user-avatar.js

```diff
@@ -218,6 +218,20 @@ import '../nuxeo-icons.js';
       this.$.character.style.textShadow = this.textShadow;
     }
 
+    __generateHue(userID) {
+      if (userID) {
+        const hue = () => {
```

Comment:
Do we need to have this as a separate function, just to call it right afterwards?

Reply:
Indeed.
Also, since it is a private function, we may not need to pass userID, that is basically created only for this function and it is based on other private function, that calls a property. We may want to simplify this.